packages/core/src/modules/oob/repository/OutOfBandRecord.ts

```diff
@@ -67,14 +67,20 @@ export class OutOfBandRecord extends BaseRecord<TagsBase> {
       .reduce((acc, curr) => [...acc, ...curr], [])
   }
 
+  public assertRole(expectedRole: OutOfBandRole | undefined) {
```

Comment:
Why can expectedRole be undefined?

Reply:
I copy-pasted from `connectionRecord` where I put it and left it by accident I guess :) Updated in both places.